src/sinks/aws_s_s/sns/mod.rs

```diff
@@ -1,3 +1,6 @@
+#![cfg(all(test, feature = "aws-sns-integration-tests"))]
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏This should actually go right above the line `mod integration_tests;`. The way the macro works is it gates the scope of the module it is declared over.